### ISSUE TITLE
Canary plug in is breaking the styling in Markdown 

### ISSUE THREAD

**Issue by Duke-Box** [jcwillox/lovelace-canary#1]
Hi,

I have a Markdown card with a config of:

'''
    cards:
      - type: markdown
        style: |
          ha-card {
            padding: 31px;
          }
          h1 {
            text-align: center;
            font-size: 90px;
          }
          h2 {
            text-align: center;
            color: var(--primary-color);
          }
        content: |
          # {{ states('sensor.local_time')}} 
          ## {{ states('sensor.local_date')}}
'''
which gives this:

![B4_Canary](https://user-images.githubusercontent.com/38861692/77009373-b3593900-695f-11ea-8bed-d5c6ecf7ed45.PNG)

When I install Canary and load the module in the lovelace YAML, the Markdown looks like this:

![After_Canary](https://user-images.githubusercontent.com/38861692/77009804-780b3a00-6960-11ea-8241-39ce9a346967.PNG)

It appears that Canary is affecting the CSS styling in the Markdown card.

I am running Home Assistant (Hassio) 106.6

Thanks 
Mike

**Comment by jcwillox** [comment]
Hey, based on the configuration above Canary will never touch the markdown card.
What are you using to style the markdown card, the core [markdown card](https://www.home-assistant.io/lovelace/markdown/) does not support styling?

Also, to format your code use ``` three backticks.

**Comment by bieniu** [comment]
I have the same issue but with other cards (sensor, entities, glance). Most of my styling from `card-mod` don't work when `canary` is loaded. For example:
```yaml
             - type: sensor
                entity: sensor.airly_info
                style: |
                  ha-card {
                    background:
                      {% set airly = states("sensor.airly_info") %}
                      {% if airly == "bardzo dobra" %}
                        #43a047
                      {% elif airly == "dobra" %}
                        #fbc02d
                      {% elif airly == "średnia" %}
                        #fb8c00
                      {% elif airly == "zła" %}
                        #f4511e
                      {% elif airly == "bardzo zła" %}
                        #d81b60
                      {% else %}
                        #000000;
                      {% endif %};
                    color: #fff;
                  }
                  ha-icon {
                    color: #ffffff;
                  }
```

**Comment by jcwillox** [comment]
Right, I think there may be a conflict issue between Canary and `card-mod`, it's certainly possible, ill have a look into it.

**Comment by jcwillox** [comment]
So I've checked it out and indeed Canary will override some of card-mods functions and vice versa, it depends on which plugin loads first. I've got a fix working which will stop Canary from overriding `card-mod` and instead work correctly alongside it. But currently if `card-mod` loads after Canary it will override Canary's functions. I will likely submit a PR to card-mod to fix this.

**Comment by Duke-Box** [comment]
> Hey,
> Based on the configuration above Canary will never touch the markdown card.
> What are you using to style the markdown card, the core [markdown card](https://www.home-assistant.io/lovelace/markdown/) does not support styling?
> 
> Also, to format your code use ``` three backticks.

**Comment by Duke-Box** [comment]
Hi,

Although I do have card-mod installed my issue was with the native markdown card. And yes this card does support styling because as of  version 0.99 all functionality from "useful markdown card" and then "markdown-mod card" was integrated into the native card.
See this

https://github.com/thomasloven/lovelace-markdown-mod

So is this something else, nothing to do with with card-mod?

Sorry about the config code, I thought I had used three backticks (see above and below my config) but I obviously used apostrophies instead. doh!

thanks

**Comment by jcwillox** [comment]
I'm pretty sure he meant all except styling which is now best done with card-mod. If you want this is the [code](https://github.com/home-assistant/frontend/blob/dev/src/panels/lovelace/cards/hui-markdown-card.ts) for the native markdown card which does not support styling, also why there's no reference to it in the [documentation](https://www.home-assistant.io/lovelace/markdown/). 

Release [`0.2.2`](https://github.com/jcwillox/lovelace-canary/releases/tag/0.2.2) will fix this issue, currently, the only limitation is that Canary must be added to your Lovelace resources after `card-mod`. If you are on a version below 0.107.0 just make sure card-mod is before canary in the resources list. On versions 0.107.0 and above you will need to remove canary from your resources and then re-add it.

**Comment by Duke-Box** [comment]
Ok but it works in the markdown card as I showed with my screenshot. If you
look at the useful markdown card documentation, at the bottom of the read
me, you will see the code I used. I just copied and pasted it into the
native markdown card using the UI. I didn't need to edit the lovelace yaml
at all. I did have to set up the sensors though and change the syntax to
jinja template in the last 2 lines.
But maybe it works because I have card mod as well, although my
understanding of the documentation made me believe otherwise.

I look forward to your fix.
Thanks
